src/components/network/NetworkActions.tsx

```diff
@@ -96,7 +96,7 @@ const NetworkActions: React.FC<Props> = ({
 
   const mineAsync = useMiningAsync(network);
 
-  const handleClick: MenuProps['onClick'] = useCallback(info => {
+  const handleClick: MenuProps['onClick'] = useCallback((info: { key: any }) => {
```

Comment:
nit: try to minimize the use of `any` when possible. Using `string` should work fine here.